Marlin/language_en.h

```diff
@@ -120,6 +120,15 @@
 #ifndef MSG_MOVE_E
 #define MSG_MOVE_E                          "Extruder"
 #endif
+#ifndef MSG_MOVE_E1
+#define MSG_MOVE_E1                         "Extruder1"
+#endif
+#ifndef MSG_MOVE_E2
+#define MSG_MOVE_E2                         "Extruder2"
+#endif
+#ifndef MSG_MOVE_E3
+#define MSG_MOVE_E3                         "Extruder3"
+#endif
```

Comment:
But, rather than having the #if EXTRUDERS < 2 in every language, you should define it like this in the default (en) file.

#ifndef MSG_MOVE_ONLY_E
  #define MSG_MOVE_ONLY_E    "Extruder"
#endif 
#ifndef MSG_MOVE_E
  #define MSG_MOVE_E    "Extruder0"
#endif 
#if EXTRUDERS <2
  #define MSG_MOVE_E0  MSG_MOVE_ONLY_E
#else 
  #define MSG_MOVE_E0 MSG_MOVE_E
#endif